src/item.cpp

```diff
@@ -7681,6 +7681,9 @@ std::set<ammotype> item::ammo_types( bool conversion ) const
         }
     }
 
+    if( is_gun() ) {
+        return type->gun->ammo;
```

Comment:
If there are multiple of these, and only some overlap with contents.ammo_types(), should only the overlapping one(s) be returned?